packages/material-renderers/src/complex/MaterialEnumArrayRenderer.tsx

```diff
@@ -38,9 +48,35 @@ export const MaterialEnumArrayRenderer = ({
   handleChange: _handleChange,
   ...otherProps
 }: ControlProps & OwnPropsOfEnum & DispatchPropsOfMultiEnumControl) => {
+  const [focused, onFocus, onBlur] = useFocus();
+  const isValid = errors.length === 0;
+  const appliedUiSchemaOptions = merge({}, config, otherProps.uischema.options);
+  const showDescription = !isDescriptionHidden(
+    visible,
+    description,
+    focused,
+    appliedUiSchemaOptions.showUnfocusedDescription
+  );
+
   return (
     <Hidden xsUp={!visible}>
-      <FormControl component='fieldset'>
+      <FormControl
+        component='fieldset'
+        fullWidth={!appliedUiSchemaOptions.trim}
+        onFocus={onFocus}
+        onBlur={onBlur}
+      >
+        <FormLabel
+          htmlFor={id}
+          error={!isValid}
+          component={'legend' as 'label'}
```

Comment:
I don't think you need to `as 'label'`. I guess this might have only been necessary in an older mui version with stricter types.
```suggestion
          component={'legend'}
```

Reply:
It's taken as-is from MaterialRadioGroup. Should I remove it also there ?

Reply:
Sure that would be great. Thanks!

Reply:
Also removed from MaterialSliderControl

Reply:
And from this line from MaterialRadioGroup `component={'fieldset' as 'div'}`

Reply:
Fixed in commit [20dceb0](https://github.com/eclipsesource/jsonforms/pull/2297/commits/20dceb010e9a1b72be1b570bc97949867836b4e2)
(Unresolved to be sure it's ok for you)